src/layouts/base-new/base-new-layout.module.css

```diff
@@ -1,9 +1,4 @@
-.baseNewLayout {
-  display: flex;
-  flex-direction: column;
-  min-height: 100vh;
```

Comment:
Have used `Min100Layout` as it implements this same functionality. Added `.header` class to ensure header bars are sticky in this layout.